godel_surface_detection/src/detection/surface_detection.cpp

```diff
@@ -102,71 +98,13 @@ void SurfaceDetection::clear_results()
 	meshes_.clear();
 }
 
-bool SurfaceDetection::load_parameters(std::string node_ns)
+bool SurfaceDetection::load_parameters(const std::string& filename, const std::string& ns)
 {
-	return load_parameters(params_,node_ns);
+	return true;
```

Comment:
I want to leave the stumps for the load/save routines so I can replace them shortly.